### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by piotrgajow**
Add package.json, config file and setup instructions in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # jira-tools
 Set of tools for extracting data from Jira issues
+
+## Usage
+
+### Setup
+
+- Run `npm install`
+- Generate authentication token
+  - Create API token on this [webpage](https://id.atlassian.com/manage/api-tokens) (Create API token > Enter label > Create > Copy to clipboard)
```
Comment (Zontey) [inline README.md]: test comment

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+  - Prepare authentication data from your Jira account email address and the API token - `email:token`
+  - Encode the authentication data using Base64 e.g. `npm start -- encode test@example.com:AbCdEfG12`
+  - Last printed line is the authentication token
+- Copy `config.template.json` to `config.json`
```
Comment (Zontey) [inline README.md]: test2

File: README.md (hunk 1 above)
Comment (piotrgajow) [reply README.md]: Won't fix

File: README.md (hunk 2 above)
Comment (piotrgajow) [reply README.md]: Won't fix

**Pull request merged**